.drone.star

```diff
@@ -484,23 +490,21 @@ def owncloudService():
         ],
     }]
 
-def testMiddleware():
+def startTestMiddleware():
+    environment = {
+        "BACKEND_HOST": "http://owncloud",
+        "NODE_TLS_REJECT_UNAUTHORIZED": "0",
+        "MIDDLEWARE_HOST": "testmiddleware",
+    }
```

Comment:
try putting the remote upload dir too....check if works
```suggestion
        "REMOTE_UPLOAD_DIR": "/usr/src/app/filesForUpload",
    }
```

Reply:
I think I tried that already. But lets try it again.

Reply:
It didn't work https://drone.owncloud.com/owncloud/client/10590/5/2